view/templates/notifications/notification.tpl

```diff
@@ -1,4 +1,29 @@
 
+<style>
+	/* Flexbox layout to align the icon and text in a single line */
```

Comment:
Please add styles in the general css file.

Reply:
The separation has taken place. I hope it's true.

Reply:
Oh yes. Thank you. I had overlooked that.